xls/tools/run_timing_characterization.py

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright 2020 The XLS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the timing_characterization with OpenROAD flow-scripts tooling and libraries."""
+# Assume that https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts are build
+# ALSO assume that OpenROAD has pre-process libraries
+import subprocess
+import datetime
+import random
+import os
+
+from absl import app
+from absl import logging
+from absl import flags
+from time import sleep
+
+
+class WorkerConfig:
+    """Hosts all configs for workers."""
+    bazel_path: str
+    openroad_path: str
+    debug: bool
+    target: str
+    yosys_bin: str
+    synthesis_libraries: str
+    sta_bin: str
+    sta_libraries: str
+    port: int
+    server_extra_args: str
+    client_extra_args: str
+
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('bazel_path', 'none', 'Root directory of bazel-bin')
```

Comment:
Instead of `'none'` use `None`.

nits:
perhaps `bazel_bin` is a better flag name?

(annoyingly) capitalize first word and end flag description (like a sentence).